index.js

```diff
@@ -95,7 +113,7 @@ function MinkeLite(config) {
 
   this.config.dir_path = this.config.dir_path || "./"
   this.config.db_name = this.config.in_memory ? ":memory:" : (
-    this.config.db_name || "minkelite.db" );
+    this.config.db_name || '' );
```

Comment:
`If the filename is an empty string, then a private, temporary on-disk database will be created. This private database will be automatically deleted as soon as the database connection is closed.`  https://www.sqlite.org/c3ref/open.html

If it doesn't get closed, if node exist with SIGTERM for example, what then?